### PR TITLE
Fix exception handling for app shutdown.

### PR DESCRIPTION
**Pull request by jmchilton**
This was broken by @jmchilton in #4910 (09c9f27f69ae72cac34b1286dfefce125ac404a6) and spotted by @nsoranzo.

### CODE REVIEW

File: lib/galaxy/app.py [hunk 1]
```diff
@@ -248,12 +248,12 @@ def shutdown(self):
         try:
             try:
```
Comment (nsoranzo) [inline lib/galaxy/app.py]: I don't think you need to nest 2 `try`, this should be enough:
```
try:
    ...
except AttributeError:
    ...
except Exception as e:
    ....
```

**Comment by dannon** [comment]
Just use two excepts instead of the nested block; should be cleaner.  Sorry I didn't spot this yesterday.

**Comment by jmchilton** [comment]
I actually switched to an ``if`` so I don't suppress ``AttributeError``s inside of ``shutdown``.

**Comment by dannon** [comment]
Digging a little more, when will that ever not exist?  Is that cruft from when I first implemented the GalaxyQueueWorker?

**Comment by nsoranzo** [comment]
@dannon Commits: 5f3c681ac4954deba0d7c6d7b473a709c031afad and 8f142f62df0b021141d2c772a73c699d12382995

@jmchilton If you don't mean to suppress the `AttributeError` any more, than you can also remove the `if`, it's redundant since the exception raised is the same with and without it.

**Comment by dannon** [comment]
@nsoranzo I saw that history, too -- it still isn't clear when the attribute will actually not exist.  (not that it matters much at all; it's exception wrapped and we should just say "Queue worker failed to shut down, or was previously terminated" or something in a single message and be done with it)

**Comment by jmchilton** [comment]
I assumed the attribute was ``self.control_worker`` - so it would make sense to have the ``if`` if we only sometimes had a control_worker and didn't want to print an exception when that was missing during shutdown but some random exception in shutdown that bubbled up we would want to print. In reality though it looks like control worker is set even before these other attributes - so it doesn't seem like the ``if`` is needed for that reason.

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**